server/src/test/java/com/epam/aidial/core/server/ResourceBaseTest.java

```diff
@@ -117,6 +117,7 @@ void init() throws Exception {
             redis = RedisServer.newRedisServer()
                     .port(16370)
                     .bind("127.0.0.1")
+                    .onShutdownForceStop(true)
```

Comment:
please leave a comment why it is needed

Reply:
That is why tests on Windows take about 6 hours to complete.

Reply:
done